src/client/ns_turn_msg.c

```diff
@@ -96,8 +96,12 @@ int stun_method_str(uint16_t method, char *smethod)
 long turn_random(void)
 {
 	long ret = 0;
-	if(!RAND_bytes((unsigned char *)&ret,sizeof(ret)))
+	if (!RAND_bytes((unsigned char*)&ret, sizeof(ret)))
+#if defined(WINDOWS)
+		ret = rand();
```

Comment:
rand() and random() need to be seeded before being used
Maybe for other PR

Reply:
There is no need to go to other PR, in this PR it is a separate commit.